android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -248,14 +253,72 @@ class RowndClient constructor(
         }
     }
 
+    private fun runRememberedRequestSignIn() {
+        rememberedRequestSignIn?.let {
+            it()
+            rememberedRequestSignIn = null
+        }
+    }
+
+    private fun isAppConfigLoadingWithCallback(callback: () -> (Unit)): Boolean {
+        val scope = CoroutineScope(Dispatchers.IO)
+        val isLoading = state.value.appConfig.isLoading
+        if (isLoading) {
+            scope.launch {
+                store.stateAsStateFlow().collect() {
+                    // Callback when the appConfig has loaded
+                    if (!it.appConfig.isLoading) {
+                        callback()
+                        scope.cancel()
+                        return@collect
+                    }
+                }
+            }
+        }
+        return isLoading
+    }
+
+    private fun isGoogleOneTapRequested(): Boolean {
+        var google = state.value.appConfig.config.hub.auth.signInMethods.google
+        return google.enabled && google.oneTap.mobileApp.autoPrompt && google.oneTap.mobileApp.delay == 0 && !hasDisplayedOneTap
```

Comment:
Throwing out a new name suggestion for this function that more closely aligns with the evaluated conditional: `isOneTapRequestedAndNotDisplayedYet()`. It may not be a perfect name, so feel free to use something better, but I think it would help readability of calling code

Reply:
Agreed. Updated